### PR TITLE
release/v1.5.10

### PR DESCRIPTION
**Pull request by bang9**
### Fix:
- Fixed a bug where the widget was not working when `configureSession` was set.
### Chore:
- Removed a `@tanstack/react-query` dependency

### CODE REVIEW

**Comment by sendbird-sdk-deployment** [comment]
[Creating Ticket] 🔖 Done https://sendbird.atlassian.net/browse/SDKRLSD-1291

File: .github/workflows/package-publish.yml [hunk 1]
```diff
@@ -49,7 +49,8 @@ jobs:
           git config --global user.email "sha.sdk_deployment@sendbird.com"
       - name: Install and Build
         run: |
-          yarn install --immutable --immutable-cache
+          yarn install
+          git diff
```
Comment (AhyoungRyu) [inline .github/workflows/package-publish.yml:53]: print 용인가요? ㅎㅎ

**Review by AhyoungRyu — APPROVED** [review]

File: .github/workflows/package-publish.yml (hunk 1 above)
Comment (bang9) [reply .github/workflows/package-publish.yml:53]: 네 에러가 나는데,, diff 는 암것두 안찍히네요...

**Pull request merged**